migrations/20151119113256_community_slackhook.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('community', function(table) {
+    table.text('slack_hook');
```

Comment:
interesting note -- i was going to suggest that `string` is more appropriate here than `text`, but then i looked it up and in Postgres [there's actually no difference](http://www.postgresql.org/docs/8.4/static/datatype-character.html). so carry on :)